Documentation/Books/Manual/ReleaseNotes/UpgradingChanges34.md

```diff
@@ -0,0 +1,25 @@
+Incompatible changes in ArangoDB 3.3
+====================================
+
+It is recommended to check the following list of incompatible changes **before**
+upgrading to ArangoDB 3.4, and adjust any client programs if necessary.
+
+The following incompatible changes have been made in ArangoDB 3.4:
+
+
+Client tools
+------------
+
+
+REST API
+--------
+
+- GET /_api/aqlfunction/ was migrated to the general way of ArangoDB Replies. 
+    It now returns an object: 
```

Comment:
returns an object => returns an object with a "result" attribute that contains the list of available AQL user functions.
In previous versions, this REST API returned only the list of available AQL user functions on the top level of the response.